### PR TITLE
feature#27-Navigation-between-screens

### PR DESCRIPTION
**Pull request by Leonardo-Aguirre-Ponce**
Co-authored-by: victordoom <23017227+victordoom@users.noreply.github.com>
Co-authored-by: Leonardo-Aguirre-Ponce <Leonardo-Aguirre-Ponce@users.noreply.github.com>
Co-authored-by: Rodrigo Aparicio <Zuack55@users.noreply.github.com>

# Description
### What?
- We made the connection between the sign up and login screens and my flights. See Initializing the React Native project #27  for more information
### Why
- To enable users to navigate between all screens. See Initializing the React Native project #27 for more information.

## Testing

- Since the project is just initialized you only need install the basic packages to visualize the application
```npm install```
- then you can initialize with an android emulator or via web
```npm start```

## Screenshots

| Changes applied |
|----------|
| image |

### CODE REVIEW

**Review by IvanGodinez21 — APPROVED** [review]
LGTM, but you should use only functions or classes for creating components or screens, I guess... 🤔

**Review by McFly1208 — APPROVED** [review]
LGTM

File: src/views/MyFlights.jsx [hunk 1]
```diff
@@ -5,99 +5,102 @@ import CardFlight from '../components/CardFlight';
...
+                  countryCode: 'BEG',
+                },
+                arrival: {
+                  country: 'Netherlands',
+                  countryCode: 'AMS',
+                },
+                date: 'September 3, 2020',
+                passengers: '2 passengers',
+              }}
+            />
```
Comment (YilKanda) [inline src/views/MyFlights.jsx:100]: What do you think about looking for an alternative to not copy the same code in your view?

Comment (QuackDuster) [reply src/views/MyFlights.jsx:100]: This is going to be changed, it was made that way with the only purpose of match the proposed design, at the moment to implement with real accounts and the database only will display the flights assigned or created for that account.

Comment (cfcortes) [reply src/views/MyFlights.jsx:100]: Taking that into consideration, i am approving this PR

**Pull request merged**